### PR TITLE
New Snap Cardano Wallet

### PR DESCRIPTION
**Pull request by khanti42**
Closes: #658

### CODE REVIEW

File: src/registry.json [hunk 1]
```diff
@@ -2596,6 +2596,42 @@
...
+      "id": "npm:@nufi/cardano-metamask-snap",
+      "metadata": {
+        "name": "Cardano Wallet",
+        "author": {
+          "name": "NuFi",
+          "website": "https://nu.fi/"
+        },
+        "website": "https://nu.fi/metamask/",
+        "summary": "Discover the Cardano universe directly from your Metamask wallet. Seamlessly store, buy, and manage ADA, Cardano tokens, and NFTs, and access a wide range of DeFi applications.",
+        "description": "1. After installing the Snap, visit wallet.nu.fi to connect your Metamask to the extended wallet. \n2. In the DApp directory, you'll find a list of Cardano dApps that support Metamask. \n3. Enjoy!",
```
Comment (FrederikBolding) [inline src/registry.json:2609]:
```suggestion
        "description": "1. After installing the Snap, visit wallet.nu.fi to connect your MetaMask to the extended wallet. \n2. In the DApp directory, you'll find a list of Cardano dApps that support MetaMask. \n3. Enjoy!",
```

File: src/registry.json [hunk 2]
```diff
@@ -2596,6 +2596,42 @@
...
+    "@nufi/cardano-metamask-snap ": {
+      "id": "npm:@nufi/cardano-metamask-snap",
+      "metadata": {
+        "name": "Cardano Wallet",
+        "author": {
+          "name": "NuFi",
+          "website": "https://nu.fi/"
+        },
+        "website": "https://nu.fi/metamask/",
+        "summary": "Discover the Cardano universe directly from your Metamask wallet. Seamlessly store, buy, and manage ADA, Cardano tokens, and NFTs, and access a wide range of DeFi applications.",
```
Comment (FrederikBolding) [inline src/registry.json:2608]:
```suggestion
        "summary": "Discover the Cardano universe directly from your MetaMask wallet. Seamlessly store, buy, and manage ADA, Cardano tokens, and NFTs, and access a wide range of DeFi applications.",
```

File: src/registry.json [hunk 3]
```diff
@@ -2596,6 +2596,42 @@
         }
       }
     },
+    "@nufi/cardano-metamask-snap ": {
```
Comment (FrederikBolding) [inline src/registry.json:2599]: The curve that this Snap needs is not in production yet. We can either approve it as hidden for now or we can wait with approval until it is in production. We don't wanna surface non-functional Snaps in the Directory.

Comment (khanti42) [reply src/registry.json:2599]: I remove the category field to potentially approve as hidden. I will check with nu.fi team to see what they prefer. 

Do you know what is the expected time of arrival of the feature they need in production ?

File: src/registry.json [hunk 4]
```diff
@@ -2596,6 +2596,41 @@
...
```
Comment (FrederikBolding) [inline src/registry.json:2599]:
```suggestion
    "@nufi/cardano-metamask-snap": {
```

**Review by FrederikBolding — APPROVED** [review]

**Review by hmalik88 — APPROVED** [review]

**Pull request merged**